final_task/rss_reader/tools.py

```diff
@@ -0,0 +1,14 @@
+import logging
+from copy import deepcopy
+
+
+def merge_lists(list1, list2):
+    """ Merge two lists """
+    logging.info('Merging lists')
+    res_list = deepcopy(list1)
+
+    for element in list2:
+        if element not in res_list:
+            res_list.append(element)
+
+    return res_list
```

Comment:
If you need resulting list to include both `list1` and `list2` without repetitions, it is better to use `set` for it. You can just simply get `list` from a `set` afterwards. Of course, in this case you will need to mind the order of the elements

Reply:
I cannot convert list of Item to set, because Item is unhashable.

Reply:
Ah, yes, I forgot that Item has attribute of list type, my bad.

Nevrmind then